### PR TITLE
Feature shuffle array

### PR DESCRIPTION
**Pull request by rashi07dashore**
- [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Java/blob/master/CONTRIBUTING.md).
- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] All filenames are in PascalCase.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] All new code is formatted with `clang-format -i --style=file path/to/your/file.java`

### CODE REVIEW

File: src/main/java/com/thealgorithms/misc/ShuffleArray.java [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            arr[i] = arr[j];
+            arr[j] = temp;
+        }
+    }
+    /**
+     * This method takes user input to create an array.
+     *
+     * @return the array created from user input
+     */
+    public static int[] getUserInputArray() {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/misc/ShuffleArray.java:41]: Please replace user input and main with JUnit tests

**Review by siriak — CHANGES_REQUESTED** [review]

File: src/main/java/com/thealgorithms/misc/ShuffleArray.java (hunk 1 above)
Comment (rashi07dashore) [reply src/main/java/com/thealgorithms/misc/ShuffleArray.java:41]: yes working on it!

Comment (rashi07dashore) [reply src/main/java/com/thealgorithms/misc/ShuffleArray.java]: > Please replace user input and main with JUnit tests

@siriak please check :) 
Thanks!!

File: src/main/java/com/thealgorithms/misc/ShuffleArray.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+     * This method prints the elements of the array.
+     *
+     * @param arr is the input array to be printed
+     */
+    public static void printArray(int[] arr) {
+        for (int num : arr) {
+            System.out.print(num + " ");
+        }
+        System.out.println();
+    }
```
Comment (siriak) [inline src/main/java/com/thealgorithms/misc/ShuffleArray.java:45]: Please also remove this method, we are not concerned with visualization

File: src/main/java/com/thealgorithms/slidingwindow/MinSumKSizeSubarray.java
Comment (siriak) [inline src/main/java/com/thealgorithms/slidingwindow/MinSumKSizeSubarray.java:1]: Do you also want to add this file? If so, you also needs tests for it

**Review by siriak — CHANGES_REQUESTED** [review]

File: src/main/java/com/thealgorithms/misc/ShuffleArray.java (hunk 2 above)
Comment (rashi07dashore) [reply src/main/java/com/thealgorithms/misc/ShuffleArray.java]: > Please also remove this method, we are not concerned with visualization

@siriak please check. I have made the changes. Thanks :))

**Comment by rashi07dashore** [comment]
@siriak Please check. Thanks :))

**Review by siriak — APPROVED** [review]
Looks good, thanks!

**Pull request merged**